src/ngraph/validation_util.hpp

```diff
@@ -114,6 +114,32 @@ namespace ngraph
     ///            by adding tensor_rank to axis.
     int64_t normalize_axis(const Node* node, std::int64_t axis, std::int64_t tensor_rank);
 
+    /// \brief      Handle out of range axes in vector.
+    ///
+    /// \param[in]  node_description  The name of node with requested axes.
+    /// \param[in]  axes              The requested vector of axes.
+    /// \param[in]  tensor_rank       The corresponding tensor rank.
+    ///
+    /// \return     If any negative axis in vector, it counts from the last to the first
+    ///             axis, by adding tensor_rank to axis.
+    ///
+    std::vector<size_t> normalize_axes(const std::string& node_description,
+                                       std::vector<int64_t> axes,
+                                       std::int64_t tensor_rank);
+
+    /// \brief      Handle out of range axis.
+    ///
+    /// \param[in]  node_description   The node with requested axis.
+    /// \param[in]  axis               The requested axis value.
+    /// \param[in]  tensor_rank        The corresponding tensor rank.
+    ///
+    /// \return    Checking if axis is in range [-tensor_rank, tensor_rank-1], otherwise
+    ///            returns error. If negative axis, it counts from the last to the first axis,
+    ///            by adding tensor_rank to axis.
+    int64_t normalize_axis(const std::string& node_description,
```

Comment:
I think that this function is unnecessary here in `ngraph` namespace since it is used only by ONNX Importer.

Reply:
As before, we want to have all `normalize` methods in one place